kong/db/strategies/cassandra/init.lua

```diff
@@ -519,44 +506,43 @@ function _mt:insert(entity, options)
   local args = new_tab(#schema.fields, 0)
   local ttl = schema.ttl and options and options.ttl
 
-  local cql, err
-  if ttl then
-    cql, err = get_query(self, "insert_ttl")
-    if err then
-      return nil, err
-    end
+  local partitioned, err = is_partitioned(self)
+  if err then
+    return nil, err
+  end
 
-    cql = fmt(cql, ttl)
+  local cols = {}
+  local binds = {}
```

Comment:
Please initialize these variables below the `args` local declaration? And we should use the `new_tab(fields_len)` allocation (caching the `#schema.fields` value), even if we may end up using less slots that allocated.